src/js/components/reviewData/ReviewDataContent.jsx

```diff
@@ -180,6 +194,11 @@ export default class ReviewDataContent extends React.Component {
                                         Monthly submissions cannot be certified
                                     </div>
         }
+        if (this.checkAffiliations() && !this.props.session.admin) {
+            certifyButtonText = "You Do Not Have Proper Permissions";
```

Comment:
Is this text cleared with Kaitlin or do we want something more like "You Do Not Have Permission To Certify" or something?